contract/r/gnoswap/gnft/gnft.gno

```diff
@@ -82,6 +82,7 @@ func MustOwnerOf(tid grc721.TokenID) address {
 // Only callable by position contract.
 func SetTokenURI(cur realm, tid grc721.TokenID, tURI grc721.TokenURI) (bool, error) {
 	halt.AssertIsNotHaltedPositionWithVersion(1)
```

Comment:
@jinoosss It seems we should also change `gnft` to use `AssertIsNotHaltedPosition` instead of the `WithVersion` function. Could you check on that?